sdk/digitaltwins/Azure.DigitalTwins.Core/tests/DigitalTwinTests.cs

```diff
@@ -121,5 +121,37 @@ public void DigitalTwins_TwinNotExist_ThrowsNotFoundException()
             act.Should().Throw<RequestFailedException>()
                 .And.Status.Should().Be((int)HttpStatusCode.NotFound);
         }
+
+        [Test]
+        public async Task DigitalTwinOperationsWithCustomObjectSerializer_Succeeds()
+        {
+            // arrange
+
+            DigitalTwinsClientOptions options = new DigitalTwinsClientOptions
+            {
+                Serializer = new TestObjectSerializer()
```

Comment:
What does `TestObjectSerializer` do?

Reply:
I see the impl below now. I'm curious what we're validating.

Reply:
Nothing special, other than object to stream and stream to object using JsonObjectSerializer internally.

Reply:
We are trying to validate that we are able to pass a custom serializer and use it through the ADT client.